### ISSUE TITLE
Blade not running when choosing No Starter Kit on fresh install

### ISSUE THREAD

**Issue by SignetPlanet** [laravel/framework#52978]
### Laravel Version

11.25.0

### PHP Version

8.3.12

### Database Driver & Version

n/a

### Description

Installing fresh instance (using `laravel new appname`) where I do not need a login (or even a database), if I select `No Starter Kit` on fresh install, the default `welcome` blade template is not rendering properly. As I needed no database, but there is no option for that, I selected `sqlite`. Where the failure differs is whether I choose to run migrations or not. 

A. If I select `sqlite` and choose **NOT** to do initial migration, upon opening in browser to the default `/` route `welcome.blade.php` view, I get an `Undefined variable $component (View: ../Code/project/vendor/laravel/frameworks/src/Illuminate/Foundation/resouces/exceptions/renderer/show.blade.php` message.

B. If I instead do a fresh install only this time selecting `sqlite` and choose **TO** do the initial migration; upon opening browser to the default `/` route `welcome.blade.php` view, I get a page load without errors however, the blade engine never runs so things like `@if` or `{{ PHP_VERSION }}` are not rendered but shown as is.

### Steps To Reproduce

create a new project using `laravel new projectname` -> select `No Starter Kit` -> choose `Sqlite` -> choose `NO` to running default db migration >  creates an `Undefined variable $component` error when loading the default page (welcome.blade.php)

create a new project using `laravel new projectname` -> select `No Starter Kit` -> choose `Sqlite` -> choose `YES` to running default db migration >  displays page where no blade tags are rendered as if blade doesn't run.

**Comment by SignetPlanet** [comment]
I found an older issue where it was a PHP version issue. If I drop back to PHP 8.2.24 this works, which was unexpected because if I use a starter kit like Breeze or Jetstream, the install finishes fine with PHP 8.3.12

I get this is an odd install (no starter kit or even database needed). I created a game utility using a single page app using only Livewire a while ago - there are no sessions, logins or database. I wanted to update this app but figured as I didn't need those I'd create a fresh install with the most bare-bone options and then manually install Livewire (or perhaps Volt) and just write a cleaner version of that app.

**Comment by Jubeki** [comment]
How do you create the Application?
- `composer create-project ...`
- `laravel new ...`
- Some other way?

If yes, was version of composer / laravel do you use?
- `composer --version`
- `laravel --version´

**Comment by Jubeki** [comment]
So I did exactly what you described and had no issues.

(Except that without a migrated database, the session can not be generated and an error is thrown, which results in a rendered error page. But you described the problem that even the error page doesn't render correctly).

What Operating System are you using? If Windows: with, or without WSL2?
How do you serve the application? Herd, php artisan serve, etc.?

**Comment by SignetPlanet** [comment]
I had already updated those when I noticed it failing before I submitted the issue. I was using:
- Laravel Installer 5.8.5
- Composer version 2.7.9

I'm running on Linux Mint (21.3) and have Apache2 installed. I have dns alias set up to run identical to Valet but on linux obviously, so I just browse to projectname.test and apache serves it. I use a little script to switch php versions (basically it runs update-alternatives to chane php-cli version and then runs a2dismod on every installed version and then a2enmod on the version I'm switching too and restarts apache).

I tried it again (again using php 8.3.12) and this time it installed. I notice this time it installed Laravel 11.26.0 instead of 11.25.0 so that seems to be the only thing that changed. Perhaps just an issue with 11.25.0. I noticed 11.26 rolled back to a older version of laravel/prompts so maybe it was related to whatever reason they rolled back for.
